modules/apps/headless-apio/media-object/media-object-apio-client-test/src/testIntegration/java/com/liferay/media/object/apio/client/test/MediaObjectApioTest.java

```diff
@@ -367,6 +619,52 @@ public void testGetDocumentsInFolder() {
 		);
 	}
 
+	@Test
+	public void testGetImageFileFromImageInDocumentRepository()
+		throws IOException {
+
+		String imageFileName = "image.png";
+
+		_documentHref = ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).header(
+			"Content-Type", "multipart/form-data"
+		).multipart(
+			"binaryFile", FileTestUtil.getFile(imageFileName, getClass())
+		).when(
+		).post(
+			_documentsHref
+		).then(
+		).extract(
+		).path(
+			"_links.self.href"
+		);
+
+		String contentHref = ApioClientBuilder.given(
+		).basicAuth(
+			"test@liferay.com", "test"
+		).header(
+			"Accept", "application/hal+json"
+		).when(
+		).get(
+			_documentsHref
+		).then(
+		).extract(
+		).path(
+			"_embedded.'Liferay:Document'.find {it.title == '" + imageFileName +
+				"'}.contentUrl"
```

Comment:
Just check the URL is not empty, I think we have a integration tests that check this, right?

Reply:
👍